src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+html {
+  --header-nav-color: #00ACDC;
+}
+
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  color: #000;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding for your links instead